### ISSUE TITLE
Mobile VO Hard to check checkboxes properly. Can't flip magnet.

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/faradays-law#208]
For https://github.com/phetsims/QA/issues/581. Found on iPad 6th Gen iPadOS 14.2 with mobile VO.

When trying to check the Voltmeter and Field Lines checkboxes, VO reads out the option several times, but only one actually allows you to select the checkbox. The one that seems most likely (where the focus lands if you directly try to click the checkbox, where the VO tells you the name of the option and that it is a checkbox, and where the highlight shows up) does not allow you to select the checkbox. It is, in fact, the option before it. I found selecting the checkbox a bit difficult due to this.

Visuals: 
https://drive.google.com/file/d/1Gtv0acaWcvuRXElc86S2EZSjHEos0gOC/view?usp=sharing

**Comment by KatieWoe** [comment]
I also cant seem to press the button to switch polarity of the magnet at all.

**Comment by jessegreenberg** [comment]
This is https://github.com/phetsims/scenery/issues/1094, which I had forgotten about. That has been a pretty difficult issue to resolve. Since this release isn't going to support description, this shouldn't block its publication. I will however, add the blocks-publication label to the scenery issue to note that no sims with description should be published until fixed.
